data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg

```diff
@@ -369,7 +369,7 @@
 
         [message]
             speaker="Ro'Sothian"
-            message= _ "Leave <i>you</i> in peace?! It is your kind that won’t leave <i>us</i> in peace! Ever since you blasted dwarves built that damn road you have harassed us unceasingly! Now, it ends here! After I kill you I will split your carcass and hang it at the mouth of this pass as a warning for any other fools like you! And unlike us <i>you</i> won’t come back."
+            message= _ "Leave <i>you</i> in peace?! It is your kind that won’t leave <i>us</i> in peace! Ever since you blasted dwarves built that damn road you have harassed us unceasingly! Now, it ends here! After I kill you I will split your carcass and hang it at the mouth of this pass as a warning for any other fools like you! And, unlike us, <i>you</i> won’t come back."
```

Comment:
```suggestion
            message= _ "Leave <i>you</i> in peace?! It is your kind that won’t leave <i>us</i> in peace! Ever since you blasted dwarves built that damn road you have harassed us unceasingly! Now, it ends here! After I kill you I will split your carcass and hang it at the mouth of this pass as a warning for any other fools like you! And unlike us, <i>you</i> won’t come back."
```